src/vstart.sh

```diff
@@ -440,6 +440,7 @@ prepare_conf() {
         filestore fd cache size = 32
         run dir = $CEPH_OUT_DIR
         enable experimental unrecoverable data corrupting features = *
+$extra_conf
```

Comment:
you might want to make sure `$extra_conf` is able to override the settings in all sections. for example, the default settings of `[client]` will override that the `$extra_conf` with your change.

Reply:
@tchaikov do you expect me add a `$extra_conf` to `[client]` section? To make sure add extra config parameters to all sections.